update wd monitors after ui change (#22898)
DataDog/documentation: content/en/monitors/types/watchdog.md

To create a [Watchdog monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Watchdog*.

{{< img src="/monitors/monitor_types/watchdog/watchdog-monitor-1.png" alt="Configuring a Watchdog Monitor" style="width:80%;">}}

## Define your query
Select the scope to be alerted on by configuring (wildcards are supported):

**1. Predefined selectors**
* Environment. These values are derived from the `env` tag.
* Alert Category. Scope the monitor to a subset of Watchdog alerts.
* Team. These values are derived from the Service Catalog.

**2. Additional scoping**

> * Filter on any additional tag available on the Watchdog event
> * Group By the dimensions you want to [group notifications by][3]
> {{< img src="/monitors/monitor_types/watchdog/watchdog-monitor-2.png" alt="Configuring a Watchdog Monitor with advanced settings" style="width:80%;">}}

Reviewer: ```suggestion
* Filter on any additional tag available on the Watchdog event.
* Group By the dimensions you want to [group notifications by][3].
{{< img src="/monitors/monitor_types/watchdog/watchdog-monitor-2.png" alt="Configuring a Watchdog Monitor with advanced settings" style="width:90%;">}}
```

After your selections are made, the graph at the top of the monitor creation page displays the matching Watchdog events over the selected time frame.

### Notifications

> For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][4] page.

Reviewer: ```suggestion
For more instructions on the **Notify your team** section, see the [Notifications][4] page.
```
